scripts/histmakers/mz_dilepton.py

```diff
@@ -147,6 +153,20 @@ def build_graph(df, dataset):
             axes = [*nominal_axes, *unfolding_axes] 
             cols = [*nominal_cols, *unfolding_cols]
 
+    if isZ:
+        # gen level variables before selection
+        for obs in [
+            "massVgen", # preFSR variables
```

Comment:
Should probably define an ew_obs array since it's used multiple times

Reply:
I'm thinking if we should keep those additional histograms, I added them for some studies but in general, they are not needed.

Reply:
You might as well keep the ability to make them, but you could turn them off by default